### ISSUE TITLE
Can't hit weeds underneath puddles

### ISSUE THREAD

**Issue by K-Dynamic** [CM-14/CM-14#2392]
## Description
Puddles block weeds, can't whack them

**Reproduction**
1. Bleed over weed
2. Knife weed under puddle
3. No weed deweeding

**Screenshots**
no

**Additional context**
explosives probably work because they don't rely on melee being blocked by puddles

**Comment by Tunguso4ka** [comment]
Duplicate of #2369
